vignettes/data_protocol.Rmd

```diff
@@ -76,15 +69,27 @@ The distributions currently included in the literature search for each pathogen/
 - "ebola serial interval"
 - "influenza H7N9 onset to admission"
 
+However, these simple search phrases can return a large number of irrelevant papers. We recommend using more specific search schema depending on the search engine used. For example, if using Google Scholar a schema like:
+
+- ("Middle East Respiratory Syndrome" OR MERS) AND "onset to death" AND (estimation OR inference OR calculation)
+- (ebola OR EVD) AND "onset to death" AND (estimation OR inference OR calculation)
+
+Or if Web of Science is being used:
+
+- ("COVID-19" OR "SARS-CoV-2") AND "onset to death" AND estimat*
+
+This should refine the results to a more suitable set of literature.
+
+
 1. **Literature search engines**: we recommend using a selection of search engines to prevent one source potentially omitting papers. Recommended search sites are: Google Scholar, Web of Science, PubMed, and Scopus. Across each site we performed the same search.  
```

Comment:
I'd agree on writing it either as "we have done this" or "you should do this" without switching to avoid confusion

Reply:
Change to consistent present tense throughout. Remove advice to readers.